project/Dependencies.scala

```diff
@@ -42,6 +42,7 @@ object Dependencies {
   val jodaConvert = "org.joda" % "joda-convert" % "1.2"
   val jSoup = "org.jsoup" % "jsoup" % "1.8.3"
   val liftJson = "net.liftweb" %% "lift-json" % "2.6-RC2"
+  val macwire = "com.softwaremill.macwire" %% "macros" % "2.2.3" % "provided"
```

Comment:
@alexduf  Thanks for your reply!
Feel free to merge with your changes 😃 as this is only my opinion.
From you response I understand better the benefit which is to avoid you to write boilerplate code of constructor initialisations with dependencies correctly passed from instances, which is faire enough!